src/test/scala/com/lightbend/kafkalagexporter/InfluxDBPusherSinkTest.scala

```diff
@@ -32,37 +33,37 @@ class InfluxDBPusherSinkTest extends fixture.FreeSpec with Matchers
     test(Fixture(properties, port))
   }
 
+  def doQuery(url: String, query: String): String = {
+    val database = "kafka_lag_exporter"
+    val cmd = Seq("curl", "-G", s"$url/query", "--data-urlencode", s"db=$database", "--data-urlencode", s"q=$query")
+    val output = cmd.!!
+    return output
+  }
```

Comment:
Ideally this would just use a Java HTTP client (like akka-http). We do this when we scrape the Prometheus HTTP endpoint for other tests.

https://github.com/lightbend/kafka-lag-exporter/blob/master/src/test/scala/com/lightbend/kafkalagexporter/integration/PrometheusUtils.scala#L27-L41

Reply:
Sure, this is also updated accordingly.